crates/driver/src/tests/cases/mod.rs

```diff
@@ -39,8 +39,5 @@ pub const ETH_ORDER_AMOUNT: u128 = 40000000000000000000u128;
 pub const DEFAULT_SCORE_MIN: u128 = 2000000000000000000u128;
 pub const DEFAULT_SCORE_MAX: u128 = 500000000000000000000000000000u128;
 
-/// The default surplus fee for limit orders.
-pub const DEFAULT_SURPLUS_FEE: u128 = 0u128;
```

Comment:
Do we already have a test that verifies that the `driver` is computing `surplus_fee`s dynamically?